### PR TITLE
work day6

### PR DESCRIPTION
**Pull request by jenicaduca**

### CODE REVIEW

File: app/cards.js [hunk 1]
```diff
@@ -1,23 +0,0 @@
-
```
Comment (jenicaduca) [inline app/cards.js]: i will remove this

File: app/js/playlist-script.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+
+var parsatPlaylist=JSON.parse(playlistsJSON);
```
Comment (gcazaciuc) [inline app/js/playlist-script.js:2]: Name better, eg parsedPlaylist

File: app/js/playlist-script.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+ Song.prototype.getSongListened = function(lis){
+    this.songListened = lis;
+  };
+
+ Song.prototype.getSongAuthor = function(auth){
+   this.songAuthor = auth;
+ };
+
+function Playlist(dataPlaylist){
```
Comment (gcazaciuc) [inline app/js/playlist-script.js:57]: Put each class into it's own file

File: app/js/formular2-script.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+function verifica(){
+  var nrUsers = parseUsers.length;
+   var bool= true;
+   var emailDiv= document.getElementById("formText1").value;
+   var passDiv= document.getElementById("formText2").value;
+
+    for(var i = 0; i < nrUsers; i++)
+        {
+          if (parseUsers[i].email == emailDiv)
```
Comment (gcazaciuc) [inline app/js/formular2-script.js:19]: Put the email verification into AuthenticationServices. use ===

File: app/js/playlist-script.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  var buton = document.getElementById('afiseaza');
+  if(obj.style.visibility == 'visible')
+    {
+      obj.style.visibility = 'hidden';
+  }  else
+     {
+     obj.style.visibility = 'visible';
+  }
+
+  for (var i=0; i < parsatPlaylist[0].songs.length; i++)
```
Comment (gcazaciuc) [inline app/js/playlist-script.js:85]: This is not ok. create a function createPlaylistModels( parsedPlaylist) that create Playlist objects for each JSON object in parsedPlaylist. Call it just after the playlists are parsed

**Pull request closed without merging**